package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build": "webpack",
+    "build": "NODE_ENV=production webpack",
```

Comment:
I agree with this. We are already doing it for the [`API_URL`](https://github.com/maryvilledev/codesplain-embedded/blob/master/circle.yml#L16). Better to be consistent when possible.